src/components/regionalization/RegionalizationModal/regionalization-modal.scss

```
@@ -0,0 +1,93 @@
+@import "src/styles/scaffold";
+
+[data-modal-overlay] {
+  background-color: rgb(0 0 0 / 25%);
+}
+
+[data-regionalization-modal] {
+  position: fixed;
+  top: 30vh;
+  right: rem(24px);
+  left: rem(24px);
+  max-width: calc(var(--fs-grid-breakpoint-desktop) / 3);
+  margin: auto;
+  background-color: var(--fs-color-body-bkg);
+  border-radius: var(--fs-spacing-0);
+
+  @include media(">=tablet") {
+    width: calc(100vw / 3);
+    min-width: calc(var(--fs-grid-breakpoint-desktop) / 3);
+  }
+
+  &[data-regionalization-modal-state="in"] {
+    animation: motion-in .2s ease-in;
+  }
+
+  &[data-regionalization-modal-state="out"] {
+    animation: motion-out .2s ease-out;
```

Some equivalent tokens I found:
```suggestion
    animation: motion-in var(--fs-transition-timing) ease-in;
  }

  &[data-regionalization-modal-state="out"] {
    animation: motion-out var(--fs-transition-timing) ease-out;
```